kafka/producer/kafka.py

```diff
@@ -538,7 +538,7 @@ def _estimate_size_in_bytes(self, key, value, headers=[]):
             return LegacyRecordBatchBuilder.estimate_size_in_bytes(
                 magic, self.config['compression_type'], key, value)
 
-    def send(self, topic, value=None, key=None, headers=None, partition=None, timestamp_ms=None):
+    def send(self, topic, value=None, key=None, headers=None, partition=None, timestamp_ms=None) -> RecordMetadata:
```

Comment:
This is just adding the type annotation, right? Does it do anything else?

I don't currently work in python much, so haven't kept up with all the latest language developments.

Reply:
It is just type annotation and users need it badly :sweat_smile: 

related to this ticket https://github.com/dpkp/kafka-python/issues/2245

Reply:
Since this library supports Python 2.7, we can use type comments to annotate stuff. It would be `# type: (...) -> RecordMetadata` to the next line of `def`.

See https://peps.python.org/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code

Reply:
Done, fixed it! Thank you for the feedback

Reply:
> This is just adding the type annotation, right? Does it do anything else?
> 
> I don't currently work in python much, so haven't kept up with all the latest language developments.

Yes @jeffwidman , it allows users to quickly trough the IDE's knows whats the return type of the function, pretty useful.
I'm sorry for the delay, and I've fixed it for compatibility with Python 2.7 as suggested by @Congee